OWNERS.md

```diff
@@ -71,6 +71,14 @@ without further review.
 * Otto van der Schaaf ([oschaaf](https://github.com/oschaaf)) (oschaaf@redhat.com)
 * Tim Walsh ([twghu](https://github.com/twghu)) (walsh@redhat.com)
 
+In addition to the permanent Envoy PST, we have additional temporary
```

Comment:
Thanks for making sure our docs are up to date!  
2 minor nits:
PST -> security team?
and contributors -> envoy-setec contributors ?